### DISCUSSION TITLE
Export Sector to VVT

### DISCUSSION THREAD

**Discussion by Gegh** [bcholmes/StarTrek2d20#178] [Ideas]
It would be awesome if there would be a way to export the generated Sectors to (Foundry) VTT

**Comment by Felderburg** [comment]
You'll need to provide an exported system or sector so that BCHolmes can see what format Foundry uses for its sectors and systems.

**Comment by bcholmes** [comment]
Is the ask fundamentally about exporting the diagrams as images?
